src/posts/abstracting-the-frontend/index.md

```diff
@@ -0,0 +1,124 @@
+---
+path: '/blog/abstracting-the-front-end'
+title: 'Abstracting The Front End'
+date: '2020-03-15'
+featuredImage: './cover.jpg'
+---
+
+We so rarely talk about the front end in abstract terms. It's all _"how to x in React"_ or _"Scaling apps in my favourite framework"._
+
+Let's take a moment and talk about how we could split up our front-end concerns on a higher level,
+agnostic of any framework or technology, so that we can avoid distractions when dealing with the aforementioned challenges.
```

Comment:
What aforemented challenges?

Reply:
It's supposed to be a reference to the initial statement (_"how to x in React"_ or _"Scaling apps in my favourite framework"._). Perhaps my vocabulary is a bit cryptic.